### PR TITLE
#5692 - Add Radio Button Input Type to SelectInput

### PR DESCRIPTION
**Pull request by yoyounik**
… the InputsForm.vue component to render the radio button

This PR addresses issue #5692 by introducing a new radio button input type for the SelectInput component. This enhancement provides a more intuitive and user-friendly way to select options.

### What changes are being made and why?

---

### How the changes have been QAed?

---

### CODE REVIEW

**Review by MilosPaunovic — CHANGES_REQUESTED** [review]
Hi @yoyounik,

I can tell you right away that removing content form `InputsForm.vue` file was not required, you just needed to add radio buttons as one more option.

**Comment by yoyounik** [comment]
Hii @MilosPaunovic 
I understand now—I’ll add the radio button option without removing any existing content from InputsForm.vue I’ll make the updates and push the changes shortly.

**Comment by yoyounik** [comment]
Hii @MilosPaunovic ,
i have updated the changes accordingly now, please go through it once if you get time and let me know if there are any further changes needed.

**Comment by MilosPaunovic** [comment]
Now it's up to @Skraye, as Java is not my strong suit.

**Comment by yoyounik** [comment]
Thanks, @MilosPaunovic! I appreciate your help. I’ll reach out to @Skraye for further assistance on this.

**Comment by yoyounik** [comment]
hii @Skraye please go through my PR once you get time and let me know about any updates i should make.

**Comment by Skraye** [comment]
Hi Yoyounik, you can start by making all the changes to have a green CI, they are indicated on Github if you go on the Files Changed pages!

**Comment by yoyounik** [comment]
hii @Skraye i have updated all the changes and the checks are passed now.

**Comment by yoyounik** [comment]
Hii @MilosPaunovic @Skraye any further changes i could make ?

**Comment by yoyounik** [comment]
Hii @Skraye can you please check once

**Comment by MilosPaunovic** [comment]
@yoyounik It will be checked, don't worry, as soon as someone gets a chance to do it.

I know you're eager to get this merged, but maybe you can change this waiting with trying to close another issue from the backlog?

File: core/src/main/java/io/kestra/core/models/flows/input/SelectInput.java [hunk 1]
```diff
@@ -39,6 +39,13 @@ public class SelectInput extends Input<String> implements RenderableInput {
     @Builder.Default
     Boolean allowCustomValue = false;
 
+    @Schema(
+        title = "Indicates if the input should be rendered as a radio button group."
+    )
+    @NotNull
+    @Builder.Default
+    Boolean isRadio = false;  // New field added here
```
Comment (Skraye) [inline core/src/main/java/io/kestra/core/models/flows/input/SelectInput.java]:
```suggestion
    Boolean isRadio = false;
```

Comment not needed

File: core/src/main/java/io/kestra/core/models/flows/input/SelectInput.java [hunk 2]
```diff
@@ -65,6 +72,7 @@ public Input<?> render(final Function<String, Object> renderer) {
                 .id(getId())
                 .type(getType())
                 .allowCustomValue(getAllowCustomValue())
+                .isRadio(isRadio)  // Include isRadio in the builder
```
Comment (Skraye) [inline core/src/main/java/io/kestra/core/models/flows/input/SelectInput.java]:
```suggestion
                .isRadio(isRadio)
```

Comment not needed

**Review by Skraye — CHANGES_REQUESTED** [review]
Radio is displayed while select is displayed too : 
image

Also, please mention the original issue in your PR

**Comment by yoyounik** [comment]
Hii @Skraye i have updated the changes , please check it whenever you get the time :)

File: ui/src/components/inputs/InputsForm.vue [hunk 3]
```diff
@@ -7,12 +7,28 @@
             :required="input.required !== false"
             :prop="input.id"
         >
+            <!-- Radio Button Group Rendering -->
```
Comment (Skraye) [inline ui/src/components/inputs/InputsForm.vue]: Not necessary comment

File: ui/src/components/inputs/InputsForm.vue [hunk 4]
```diff
@@ -61,14 +75,12 @@
             <el-input
                 type="password"
                 v-if="input.type === 'SECRET'"
-                :data-test-id="`input-form-${input.id}`"
```
Comment (Skraye) [inline ui/src/components/inputs/InputsForm.vue]: Why this deletion ? Unrelated I think

File: ui/src/components/inputs/InputsForm.vue [hunk 5]
```diff
@@ -21,8 +37,7 @@
                 :full-height="false"
                 :input="true"
                 :navbar="false"
-                v-if="input.type === 'ENUM' || input.type === 'SELECT'"
-                :data-test-id="`input-form-${input.id}`"
+                v-else-if="input.type === 'ENUM' || input.type === 'SELECT'"
```
Comment (Skraye) [inline ui/src/components/inputs/InputsForm.vue]: - Put the radio group below the el-select
- Add the radio condition in both v-if of the el-select and the el-radio-group

**Review by Skraye — CHANGES_REQUESTED** [review]

**Comment by yoyounik** [comment]
@Skraye has updated the code , please check it and tell if any further changes are needed :)

File: ui/src/components/inputs/InputsForm.vue (hunk 4 above)
Comment (yoyounik) [reply ui/src/components/inputs/InputsForm.vue]: i believe this was not in the original code, so i will keep it deleted, not required actually

Comment (Skraye) [reply ui/src/components/inputs/InputsForm.vue]: After verification, this is in the original code
Please only modify what's needed

Comment (yoyounik) [reply ui/src/components/inputs/InputsForm.vue]: please check it once again, i have updated, now all the checks are running.

**Comment by yoyounik** [comment]
checked the developer main branch code and updated with the original code, please check once if any further changes are required :)

**Comment by Skraye** [comment]
That's still not good, in case of an enum/select with radio, both are displayed :
image

Also, please restore the unrelated deleted code

**Comment by MilosPaunovic** [comment]
@yoyounik If you look at files tab of this PR, you'll notice a lot of indentation errors, don't forget to handle those.

**Pull request merged**